CONTRIBUTING.md

```diff
@@ -0,0 +1,104 @@
+
+Once the feature is done you can request it to be merged back into `master` by making a Pull Request.
+
+Before making the pull request it is a good idea to rebase your branch to `master` to ensure that eventual conflicts with the `master` branch is solved before the PR is reviewed and we can therefore have a clean merge.
+
+
+
+### General stuff about git and commit messages
+
+In general it is better to commit often. Small commits are easier to roll back and also makes the code easier to review.
+
+Write helpful commit messages that describes the changes and possibly why they were necessary.
+
+Each commit should contain changes that are functionally connected and/or related. If you for example want to write _and_ in the first line of the commit message this is an indicator that it should have been two commits.
+
+Learn how to select chunks of changed files to do multiple separate commits of unrelated things. This can be done with either `git add -p` or `git commit -p`.
+
+
+#### Helpful commit messages
+
+The commit messages may be seen as meta-comments on the code that are incredibly helpful for anyone who wants to know how this piece of software is working, including colleagues (current and future) and external users.
+
+Some tips about writing helpful commit messages:
+
+ 1. Separate subject (the first line of the message) from body with a  blank line.
+ 2. Limit the subject line to 50 characters.
+ 3. Capitalize the subject line.
+ 4. Do not end the subject line with a period.
+ 5. Use the imperative mood in the subject line.
+ 6. Wrap the body at 72 characters.
+ 7. Use the body to explain what and why vs. how.
+
+For an in-depth explanation of the above points, please see [How to Write a Git Commit Message](http://chris.beams.io/posts/git-commit/).
+
+
+### How we do code reviews
+
+A code review is initiated when someone has made a Pull Request in the appropriate repo on github.
+
+Work should not continue on the branch _unless_ it is a [Draft Pull Request](https://github.blog/2019-02-14-introducing-draft-pull-requests/). Once the PR is marked ready the review can start.
+
+The initiator of the PR should recruit 2 reviewers that get assigned reviewer duty on the branch.
+
+Other people may also look at and review the code.
+
+A reviewers job is to:
+
+  * Write polite and friendly comments - remember that it can be tough to have other people criticizing your work, a little kindness goes a long way. This does not mean we should not comment on things that need to be changed of course.
+  * Read the code and make sure it is understandable
+  * Make sure that commit messages and commits are structured so that it is possible to understand why certain changes were made.
+  * Ensure that the test-suite covers the new behavior
+
+It is _not_ the reviewers job to checkout and run the code - that is what the test-suite is for.
+
+Once all the reviews are positive the Pull Request can be _merged_ into `master` and the feature branch deleted.
```

Comment:
```suggestion
Once at least 3 reviews from 3 different partners are positive, the Pull Request can be _merged_ into `main` and the feature branch deleted.

If it takes long for some partner to review code, it is common practice to try to contact them on slack to see what the problem is and if it can be resolved quickly or whether it's ok to merge.
```